.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# OS
+ARG VARIANT=bullseye
+FROM --platform=linux/amd64 mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive && apt-get install -y firefox-esr
```

Comment:
You could also use the MS provided python images as more guaranteed support being maintained since they use these in the prebuilt templates for vscode, and it allows you to encode the specific version as well (if you want, or use latest, but probably you should fix the version to avoid a random break in future).

    FROM mcr.microsoft.com/devcontainers/python:3.11